### PR TITLE
Html task 2

### PR DESCRIPTION
**Pull request by KamilWasilonek**
https://kamilwasilonek.github.io/ui-internship-lwo2/html/Task_2/

### CODE REVIEW

**Review by tkachukivan — APPROVED** [review]
Generally it would be more logic if by default you would have checkboxes selected and when user unselect it don't show some info

**Review by tsvetkovv — CHANGES_REQUESTED** [review]
It's not appropriate for production web-site
Kapture 2019-05-28 at 18 37 27

File: html/Task_2/style.css [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+  background: blue;
+}
+
+input[type="checkbox"] {
+  display: none;
+}
+
+/* Checkbox operations */
+#red:not(:checked) ~ .colors .red {
+  display: none;
```
Comment (tsvetkovv) [inline html/Task_2/style.css]: the same rule for 3 selectors. Could you union this?

Comment (tsvetkovv) [reply html/Task_2/style.css]: the same for opacity

**Review by tsvetkovv — APPROVED** [review]

**Pull request merged**